parse/functions.go

```diff
@@ -653,6 +670,17 @@ func defaultFormat(name string) FormatFunc {
 	}
 }
 
+// decimal1000 is a decimal type with a precision of 1000.
+var decimal1000 *types.DataType
+
+func init() {
```

Comment:
I feel we should always put `init` at the very top, but I couldn't find a guide for this. nbd

Reply:
Yeah I thought about it, but honestly I think it's more clear next to the decimal since it is what we are initializing.